### DISCUSSION TITLE
Issues with defining custom css

### DISCUSSION THREAD

**Discussion by sophiaas** [executablebooks/meta#234] [Q&A]
I'm following the instructions laid out [here](https://jupyterbook.org/advanced/sphinx.html#custom-css-or-javascript) for defining custom css. To test, I used the exact example given in the instructions, i.e. a file named myfile.css in folder _static using the css for justified text. 

When I build my book, the css is not getting pasted into the head of my html files. I am able to control style by manually pasting css into the header, but for some reason this is not automatically happening at build. 

Has anyone else encountered this issue? Any ideas on what might be going wrong?

My directory structure looks like this:

|── _config.yml
|── _toc.yml
|── page1.md
|── _static
    └── myfile.css

**Comment by choldgraf** [comment]
Hmmm that is weird :-/  any chance you have a link to a repository that reproduces the bug?

**Comment by lwasser** [comment]
@sophiaas @choldgraf interestingly i'm noticing an issue  locally  too with custom css. The `_static/myfile.css` file doesn't seem to always get moved / updated when you rebuild the book. i've been  manually deleting the  entire html build file and forcing a rebuild and it then seems to work. 

at first i thought my custom css was not working but t hen realized it was in  fact jupyter not moving things over. i'm curious if anyone else is seeing this. i have the most current version.

**Reply by choldgraf** [reply]
hmmm, that is pretty weird...it sometimes copies over but not always? Are there conditions under which it does/doesn't copy?

**Comment by lwasser** [comment]
update: it seems to be working with me on the current version of the book as of today Sept 2022. So this may have been resolved.
